Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM alpine:latest
 
 # install chrony
-RUN apk add --no-cache chrony
+RUN apk add --no-cache chrony && \
+    rm /etc/chrony/chrony.conf
```

Comment:
I had done it in the Dockerfile to save a wee bit of bytes. Moved it back to the `startup.sh` to keep it explicit, though it would be overwritten in there anyway.